includes/admin.form.inc

```diff
@@ -263,21 +263,33 @@ function islandora_openseadragon_admin($form, &$form_state) {
  * Validates that the supplied djatoka base URL is a working djatoka install.
  */
 function djatoka_base_url_validate($element, &$form_state, $form) {
-  $url = $element['#value'];
+  $jp2_image = drupal_get_path('module', 'islandora_openseadragon') . '/test/test.jp2';
+  $image_url = url($jp2_image, array('absolute' => TRUE));
```

Comment:
`url()` should not be used for paths to static content; use [`file_create_url()`](https://api.drupal.org/api/drupal/includes%21file.inc/function/file_create_url/7.x).